### PR TITLE
Dishonored gear patch

### PR DESCRIPTION
**Pull request by Safairette**
## Additions

Added CE compatibility to Assassin Coat and Assassin Mask from their respective mods

## Reasoning

Smoke resistance on the mask and toxin resistance on both items added as in the original lore assassins are partially resistant to poison
Additional offsets on the coat added to more closely resemble the items description

## Testing

Check tests you have performed:
- [x] Compiles without warnings
- [x] Game runs without errors
- [x] (For compatibility patches) ...with and without patched mod loaded
- [ ] Playtested a colony (specify how long)

### CODE REVIEW

File: Patches/Dishonored Assassin Coat/ThingDefs_Misc/Apparel.xml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+						<Bulk>1.5</Bulk>
+						<WornBulk>4</WornBulk>
+						<StuffEffectMultiplierArmor>6</StuffEffectMultiplierArmor>
+					</value>
+			    </li>
+				<!-- Misc -->
+				<li Class="PatchOperationAdd">
+					<xpath>Defs/ThingDef[defName="Apparel_AssassinCoatMale"]/apparel/layers</xpath>
+					<value>
+						<li>Webbing</li>
```
Comment (onigamii) [inline Patches/Dishonored Assassin Coat/ThingDefs_Misc/Apparel.xml:38]: Why are you using Webbing? I'd switch it to Shell

**Review by onigamii — COMMENTED** [review]
Also i suggest adding the wearingGaskMask hediff.

File: Patches/Dishonored Assassin Coat/ThingDefs_Misc/Apparel.xml (hunk 1 above)
Comment (N7Huntsman) [reply Patches/Dishonored Assassin Coat/ThingDefs_Misc/Apparel.xml:38]: I concur, I'd make it the same layer as the Duster.

**Comment by N7Huntsman** [comment]
> I think the mask should have the wearingGaskMask heddif applied.

I disagree, primarily due to the major debuffs associated with it.

**Comment by Safairette** [comment]
My explanations for the changes: 
- By the time you can craft the assassin mask you can already craft the full gas mask, so making the mask have full resistance + hediff would be mostly useless, as a gas mask + helmet would completely outclass it armor wise, so I opted to give it partial resistance, but without the debuff (tho I will likely need to reduce the resistances a bit more)
- The coat has a lot of extra pockets and pouches, already giving it some extra carry bulk, also those are mostly chest pouches, so wearing a vest on top would be very inconvenient, so the coat was made to occupy the same layer as a duster + block wearing webbing slots

**Review by N7Huntsman — APPROVED** [review]
Seems good to go.

**Pull request merged**